cibyl/plugins/openstack/printers/colored.py

```diff
@@ -207,6 +218,24 @@ def print_deployment(self, deployment):
             printer.add("No openstack information associated with this job", 1)
         return printer.build()
 
+    def print_test_collection(self, test_collection):
+        """Print the test collection used in an Openstack deployment.
+
+        :param test_collection: The test collection used in the deployment
+        :type test_collection: :class:`TestCollection`
+        """
+        printer = IndentedTextBuilder()
+        printer.add(self.palette.blue('Testing information: '), 0)
+        printer.add(self.palette.blue('Test suites: '), 1)
+        for test in test_collection.tests.value:
```

Comment:
Check before if test_collection.tests.value is None, you could get an error otherwise.